### DISCUSSION TITLE
Welcome to Hacktoberfest2022 Discussions!

### DISCUSSION THREAD

**Discussion by rohansaini886** [rohansaini886/Hacktoberfest2022#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rohansaini886** [comment]
Everyone Please start making Pull requests after 1st October.

**Reply by 404Nikhil** [reply]
It's showing this repository has been excluded on the official website Is it true please update us about it don't leave us hanging like this? Thank you.
